tools/hangar/src/package.setup.ts

```diff
@@ -1,6 +1,7 @@
 import { execa } from "execa";
 import * as fs from "fs-extra";
 import * as path from "path";
+import * as nodeAssert from "node:assert";
```

Comment:
ℹ️  Also added proper assertions here for prettier errors. Note that we can't use vitest assertions here because this is not an actual test.